vllm/entrypoints/llm.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         swap_space: int = 4,
         enforce_eager: bool = False,
         max_context_len_to_capture: int = 8192,
-        disable_custom_all_reduce: bool = True,
+        disable_custom_all_reduce: bool = False,
```

Comment:
We should leave it as default to False instead of True.

Reply:
Because of performance or stability?

Reply:
just checked upstream. It seems upstream has already defaulted this to False. So it is fine. Sorry, I was thinking the old behavior.